From 081b8c0560f2fe565985f9604d034918bb7acb34 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Sat, 12 Jul 2025 19:36:15 +0800
Subject: [PATCH 60/67] AOSCOS: net/phytmac: Replace xdp_do_flush_map() with
 xdp_do_flush()

Per 7f04bd109d4c ("net: Tree wide: Replace xdp_do_flush_map() with
xdp_do_flush()."):

xdp_do_flush_map() is deprecated and new code should use xdp_do_flush()
instead.

Replace xdp_do_flush_map() with xdp_do_flush().

Signed-off-by: Xi Ruoyao <xry111@xry111.site>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/net/ethernet/phytium/phytmac_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/phytium/phytmac_main.c b/drivers/net/ethernet/phytium/phytmac_main.c
index e7ce09d66adc..03392c537666 100644
--- a/drivers/net/ethernet/phytium/phytmac_main.c
+++ b/drivers/net/ethernet/phytium/phytmac_main.c
@@ -1247,7 +1247,7 @@ static int phytmac_rx(struct phytmac_queue *queue, struct napi_struct *napi,
 	}

 	if (xdp_xmit & PHYTMAC_XDP_REDIR)
-		xdp_do_flush_map();
+		xdp_do_flush();

 	phytmac_rx_clean(queue);

--
2.50.1
